packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -97,7 +98,6 @@ export class TSClient implements Generatable {
         ? (Object.keys(binaryPaths.libqueryEngine ?? {}) as BinaryTarget[])
         : (Object.keys(binaryPaths.queryEngine ?? {}) as BinaryTarget[])
 
-    const inlineSchema = await readFile(schemaPath, 'utf8')
```

Comment:
I don't understand why this read from disk were here. We are already given a `this.options.datamodel`.